modules/ROOT/pages/sftp/sftp-documentation.adoc

```diff
@@ -106,189 +110,233 @@ An FileSystemProvider which provides instances of SftpFileSystem from instances
-| File Matching Rules a| <<matcher>> |  +++A matcher used to filter the output list.+++ |  | {nbsp}
-| Target Variable a| String |  +++The name of a variable on which the operation's output is placed.+++ |  | {nbsp}
-| Target Value a| String |  +++An expression that is evaluated against the operation's output. The outcome of that expression is stored in the target variable.+++ |  +++#[payload]+++ | {nbsp}
+| Directory Path a| String | The directory path to list. |  | *x*{nbsp}
+| Recursive a| Boolean |  Whether to include the contents of sub-directories. Defaults to `false`. | `false` | {nbsp}
+| File Matching Rules a| <<matcher>> |  A matcher used to filter the output list. |  | {nbsp}
+| Time Between Size Check a| Number |  Wait time between size checks to determine if a file is ready to read. |  | {nbsp}
+| Time Between Size Check Unit a| Enumeration is one of:
+
+* NANOSECONDS
+* MICROSECONDS
+* MILLISECONDS
+* SECONDS
+* MINUTES
+* HOURS
+* DAYS |  time unit to be used in the wait time between size checks. |  | {nbsp}
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* <<non-repeatable-iterable>> |  Configure whether to use repeatable streams and their behavior. |  | {nbsp}
+| Target Variable a| String |  The name of a variable on which the operation's output will be placed. |  | {nbsp}
+| Target Value a| String |  An expression that is evaluated against the operation's output, the outcome of which will be stored in the target variable. |  `#[payload]` | {nbsp}
 | Reconnection Strategy a| * <<reconnect>>
-* <<reconnect-forever>> |  +++A retry strategy in case of connectivity errors.+++ |  | {nbsp}
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  | {nbsp}
 |===
 
 ==== Output
 [cols=".^50%,.^50%"]
 |===
-| *Type* a| Array of Message of [Binary] payload and [<<SftpFileAttributes>>] attributes
+| *Type* a| Array of Message of [Any] payload and [<<SftpFileAttributes>>] attributes.
 |===
 
-==== For Configurations
+==== For Configurations.
 * <<config>> {nbsp}
 
 ==== Throws
-* SFTP:RETRY_EXHAUSTED {nbsp}
-* SFTP:ACCESS_DENIED {nbsp}
 * SFTP:ILLEGAL_PATH {nbsp}
-* SFTP:CONNECTIVITY {nbsp}
+* SFTP:ACCESS_DENIED {nbsp}
 
 
 [[move]]
 === Move
 `<sftp:move>`
 
-Moves the file at the sourcePath into the targetPath. If targetPath doesn't exist, and neither does its parent, then an attempt is made to create depending on the value of the createParentFolder argument. If createParentFolder is false, then a SFTP:ILLEGAL_PATH error is thrown. If the target file already exists, then it is overwritten if the overwrite argument is true. Otherwise, a SFTP:FILE_ALREADY_EXISTS error is thrown. As for the sourcePath, it can either be a file or a directory. If it points to a directory, then it is moved recursively.
+
+Moves the file at the `sourcePath` into the `targetPath`. 
```

Comment:
GLOBALLY:
Change all the tick marked names into the Studio fields per the fields listed for each operation.